### PR TITLE
fix: add <ksoe-codes>

### PR DESCRIPTION
**Pull request by ksoe-codes**
resolves #issueNumber

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by ksoe-codes** [comment]
resolve #1261

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM 🚀 Thank you so much for contributing

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by ksoe-codes** [comment]
Open to contribute more :)
Greetings ✨

File: profiles/ksoe-codes.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://avatars.githubusercontent.com/u/108216630?v=4",
```
Comment (Panquesito7) [inline profiles/ksoe-codes.json:2]: Hmm, maybe you should add another picture of your profile? What do you think?


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Comment (ksoe-codes) [reply profiles/ksoe-codes.json:2]: Oh sorry but i still don't want to add a profile picture, would be happy to add it soon :))
Regards ✨

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing!


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Comment by ksoe-codes** [comment]
Ok thanks

**Pull request closed without merging**